src/lib.rs

```diff
@@ -25,7 +26,7 @@ mod tests {
         type Output = i32;
 
         fn process(&mut self, packet: Self::Input) -> Self::Output {
-            println!("Got packet {} in element {}", packet, self.id);
+            //println!("Got packet {} in element {}", packet, self.id);
```

Comment:
In any case, we shouldn't be committing commented code, but I don't think removing logging altogether is a good idea. It would be better if we could enable logging with via an environment variable, at least until we can enable `tokio_trace`.

I think something like [env_logger](https://docs.rs/env_logger/0.6.2/env_logger/) does the job quite well. That can be introduced in a separate commit. I'd be happy to look into that if you want.

Reply:
Yes that would be a good idea, leave in for now,  resolve in new commit.